WickedEngine/wiVersion.cpp

```diff
@@ -7,7 +7,7 @@ namespace wiVersion
 	// main engine core
 	const int major = 0;
 	// minor features, major updates
-	const int minor = 42;
+	const int minor = 43;
```

Comment:
That's right, these changes will be a new minor version, and revision need to set to 0 after minor version changes.